Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PG_MAJOR=15
+FROM postgres:$PG_MAJOR
+ARG PG_MAJOR
+
+COPY . /tmp/pg_embedding
+
+# Locale settings
+RUN apt-get clean && apt-get update && \
+    apt-get install -y locales && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+
+ENV LANG en_US.utf8
```

Comment:
I'm curious, Is this required for the extension, or is it just to make the image more usable?

Reply:
I'm not sure this step is required, but I got multiple errors with locale, and this fixed it.